server/db/migrations/0520181209075758_create.js

```diff
@@ -14,7 +14,7 @@ exports.up = function(knex, Promise) {
     table.string('photo', 300);
     table.string('website', 255);
     table.string('description', 1000);
-    table.integer('phone_number');
+    table.bigInteger('phone_number');
```

Comment:
Because of the variance in phone number formats they are often stored as a string rather than an integer.